### DISCUSSION TITLE
How can I configure the H5PPlayerUI and H5PEditorUI components of React?

### DISCUSSION THREAD

**Discussion by emmanuelpineda** [Lumieducation/H5P-Nodejs-library#2330] [Q&A]
How can I configure the H5PPlayerUI and H5PEditorUI components to connect directly to the h5p server?

I have two applications on different domains and servers, on server 1, I use the components mentioned above and on server 2, I have the ajax services of H5P. How can I tell the components the url of the h5p server to connect directly to it without using a proxy on server 1?

Thank you in advance.

**Comment by sr258** [comment]
Have you checked out the docs at https://docs.lumi.education/usage/rest ? The web components are connected to the server through the "Content Service". It covers the CRUD operations for content. The Ajax calls made by H5P internally must be configured through IH5PConfig object (there's `ajaxUrl` etc.). Alternatively you can pass in a custom IUrlGenerator implementation in the constructors.

It is generally possible to run the frontend and the API on different servers, but you'll have to find a way to get authentication to work in a scenario like this. This might not be trivial.
